src/modules/users/users.service.ts

```diff
@@ -20,4 +20,15 @@ export class UsersService {
     }
     return this.userRepository.save(user);
   };
+
+  getUserRoles = async (userId: number): Promise<string[]> => {
+    const user = await this.userRepository
+      .createQueryBuilder('user')
```

Comment:
can we use type orm instead of query builder? or is there any reason to use it?

Reply:
let me check it out